aruco_ros/src/marker_publish.cpp

```diff
@@ -44,9 +45,18 @@ or implied, of Rafael Muñoz Salinas.
 #include <sensor_msgs/image_encodings.h>
 #include <aruco_ros/aruco_ros_utils.h>
 #include <aruco_msgs/MarkerArray.h>
+#include <geometry_msgs/Point.h>
 #include <tf/transform_listener.h>
+#include <tf/transform_broadcaster.h>
 #include <std_msgs/UInt32MultiArray.h>
 
+string int_to_string( const int a )
```

Comment:
If it's ok to add the C++11 build flag to `CMakeLists.txt`, `std::to_string` would be the way to go.